# variables of the message

string event
float32[] param
